frontend/public/profile.html

```diff
@@ -3,9 +3,28 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Profile</title>
+	<link rel="stylesheet" href="/styles.css">
+    <title>AChat Profile</title>
 </head>
+
 <body>
-    <h1>Profile page</h1>
+    <div class="welcomeMessage" id="welcomeMessage">Profile Page</div>
+	<div><input type="button" class="chatroomButton" id="button" name="button" value="Join Chatroom" onclick="window.location.href='/chat.html'"></div>
+	<div><input type="button" class="helpButton" id="button" name="button" value="Help" onclick="window.location.href='/help.html'"></div>
 </body>
-</html>
\ No newline at end of file
+
+<script>
```

Comment:
I suggest split js script into seperate file so that it's easier later for others to work with

Reply:
I was thinking the same thing.